config/airkeeper.json.example

```diff
@@ -0,0 +1,21 @@
+{
+  "chains": [
+    {
+      "contracts": {
+        "RrpBeaconServer": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788"
+      }
+    }
+  ],
+  "triggers": {
+    "rrpBeaconServerKeeperJobs": [
+      {
+        "templateId": "0xe3214e41c4aad33825c4970025a59db034f98145f33c6bfcb8b38a6b55336584",
+        "oisTitle": "Currency Converter API",
+        "endpointName": "convertETHToUSD",
```

Comment:
Should the template also contain the `endpointName`?

Reply:
I guess this is related to this comment I just saw: https://github.com/api3dao/airkeeper/pull/2#discussion_r759959748

Reply:
Yeah, so you can find what an `endpointId` means here https://docs.api3.org/airnode/v0.2/reference/deployment-files/config-json.html#rrp

Reply:
Right but my concern is:
1. if `triggers.rrpBeaconServerKeeperJobs` will only have `templateId`
2. I can then get the `endpointId` from the template by fetching it by id
3. but I need the `endpointName` which I cannot get from the `endpointId` hash and I'm guessing that's the reason why `triggers.rrp` contains both `endpointId` and `endpointName`

This could also be solved if we add the `oisTitle` and `endpointName` to the template but I would prefer to have them in config file.

Reply:
> I'm guessing that's the reason why triggers.rrp contains both endpointId and endpointName

`triggers.rrp` has `endpointId`, `oisTitle`, `endpointName` for the node to be able to find what OIS+endpoint an `endpointId` is referring to. This is the same thing you need to be doing it here so you can use it the same way.